src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -126,17 +128,19 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
             shipping: this._getShippingAddress(),
             billing: this._getBillingAddress(),
             items: this._getItems(),
-            discounts: this._getDiscounts(),
             metadata: {
                 shipping_type: consignment.selectedShippingOption.type,
                 mode: 'modal',
+                platform_type: 'bigcommerce',
+                platform_version: '',
```

Comment:
Affirm requested the following changes

>  In our case,

      "platform_type": "BigCommerce" (mandatory)
      "platform_version": refers to the version of BigCommerce the customer is on (if applicable)
      "platform_affirm": the version of the BigCommerce/Affirm integration. If there is no versioning in place, leave blank.

Do we have the bigcommerce version available at the checkout-sdk ? otherwise I think we can leave it blank.

Reply:
I don't think we have. When you say you can leave it blank, do you mean sending an empty string or leaving it out?

Reply:
@davidchin We can remove it from the payload meanwhile. However, I sent an email to the Affirm folks asking about this to confirm.

Reply:
@davidchin Affirm confirmed that we must sent the platform_version & platform_affirm as **empty strings**